podman4: network-usage blog update install instructions (#509)
containers/podman.io: _posts/2022-02-04-network-usage.md

If you have never installed Podman, replace `upgrade` with `install` in the following command.
> $ sudo dnf upgrade podman

> If podman was upgraded, you have to install netavark explicitly otherwise the package will keep using CNI and does not install netavark.

Reviewer: ```suggestion
If Podman was upgraded, you have to install netavark explicitly otherwise, the Podman package will continue to use  CNI.
```
